packages/dds/tree/src/feature-libraries/sequence-field/fence.json

```diff
@@ -0,0 +1,5 @@
+{
+    "tags": [ "sequence-field" ],
+    "exports": [ "index" ],
+    "imports": ["forest", "tree", "util", "feature-libraries" ]
```

Comment:
We can but sadly, since there are some files we need in the "feature-libraries" folder (e.g., `treeTextCursorLegacy`) we need to add "feature-libraries" which automatically includes all of its submodules. Perhaps we can have a cleanup phase that removes such lone files and then makes more precise imports.